hypha/apply/funds/templates/funds/includes/screening_status_block-button.html

```diff
@@ -0,0 +1,96 @@
+{% load i18n heroicons %}
+
+<div class="relative"
+     x-data="{ isOpen: false, openedWithKeyboard: false, leaveTimeout: null }"
+     @mouseleave.prevent="leaveTimeout = setTimeout(() => { isOpen = false }, 250)"
```

Comment:
super nit-picky but I think I'd almost prefer no timeout with the mouse leave. To me it feels a little a little lethargic/cluttered when trying to move between the two screening options. Super small detail and may be my own personal preference though.

https://github.com/HyphaApp/hypha/assets/145372368/2bcc9185-dad6-49a9-9f5e-f6038acb05a8

Reply:
This is great! I totally get the accessibility concerns, I think you found the perfect medium at 100ms though. Seems a lot snappier.